phc/adapter.py

```diff
@@ -0,0 +1,36 @@
+import aiohttp
+
+
+class Adapter:
+    should_refresh: bool
+
+    def __init__(self):
+        self.should_refresh = True
+
+    async def send(
+        self,
+        *,
```

Comment:
I don't really care, but I'm curious why this is here to force keyword only arguments? Is there a specific concern being addressed?

Reply:
No, it was mostly just keeping what was there before. I'm ambivalent as well. 😄